docs/source/getting-started/macos/index.rst

```diff
@@ -3,98 +3,76 @@ Mac-OS Installation
-   export PATH="/usr/local/opt/gnu-sed/libexec/gnubin:$PATH"
+      sudo chown $USER /opt
 
-Install gnu-sed and make sure that it is the default ``sed`` command line application::
+   (alternatively, use a different directory).
 
-   brew install gnu-sed --with-default-names
+4. Fetch the Sming repository::
 
-If you have already installed ``gnu-sed`` but it is not the default one,
-then make sure to uninstall it and install it again with the correct options::
+      git clone https://github.com/SmingHub/Sming /opt/sming
 
-   brew uninstall gnu-sed
-   brew install gnu-sed --with-default-names
+   If using a directory other than ``/opt`` then you'll need to edit ``sming/Tools/export.sh`` before proceeding.
 
-ESP8266 Toolchain
------------------
+5. Run the installer::
 
-We pull this in from the `SmingTools <https://github.com/SmingHub/SmingTools/releases>`__ repository::
+      source /opt/sming/Tools/install.sh all
 
-   cd ~/
-   export ESP_HOME=/opt/esp-quick-toolchain
-   curl -LO https://github.com/SmingHub/SmingTools/releases/download/1.0/x86_64-apple-darwin14.xtensa-lx106-elf-e6a192b.201211.tar.gz
-   sudo mkdir -p $ESP_HOME
-   sudo tar -zxf x86_64-apple-darwin14.xtensa-lx106-elf-e6a192b.201211.tar.gz -C $ESP_HOME
-   sudo chmod -R 775 $ESP_HOME
+   If you want to save disk space then you can select which tools to install.
+   Get a list of available options like this::
 
-You can also build it yourself
-`with Homebrew <https://github.com/pfalcon/esp-open-sdk#macos>`__ or
-`with MacPorts <http://www.esp8266.com/wiki/doku.php?id=setup-osx-compiler-esp8266>`__.
+      /opt/sming/Tools/install.sh
 
-Get Sming Core
---------------
 
-Clone from the `Sming <https://github.com/SmingHub/Sming>`__ repository::
+Vscode
+------
 
-   cd <your-favourite-development-folder>/
-   git clone https://github.com/SmingHub/Sming.git
-   cd Sming
+See https://code.visualstudio.com/.
 
-.. warning::
 
-   Do NOT use the --recursive option for the command above.
-   Our build mechanism will take care to get the third-party sources and patch them, if needed.
+Eclipse
+-------
+
+*(Optional)*::
 
-You will get a copy of our `develop` branch which is intended for developers.
-It is the one where all new cool (unstable) features are landing.
+   brew install Caskroom/cask/eclipse-cpp
 
-If you want to use our stable branch then use the master branch::
 
-   git checkout origin/master
+.. note::
 
-Finally, set the :envvar:`SMING_HOME` environment variable to point to <your-favourite-development-folder>/Sming/Sming::
+   The following information may be out of date.
 
-   export SMING_HOME=`pwd`
 
 Environment Variables
 ---------------------
 
 Open with a text editor the ``.profile`` file in your home directory, and add these lines::
 
    export ESP_HOME=/opt/esp-quick-toolchain
-   export SMING_HOME=<your-favourite-development-folder>/Sming/Sming
+   export SMING_HOME=opt/Sming/Sming
```

Comment:
`export SMING_HOME=opt/Sming/Sming`

Either change this command to `export SMING_HOME=/opt/Sming/Sming` or revert this line.